.github/workflows/everything.yml

```diff
@@ -146,6 +137,48 @@ jobs:
         if: ${{ matrix.constrains != 'build_only' }}
         run: gha/scripts/ci/gh-actions/run.sh test
 
+  linux_el8:
+    needs: [format, git_checks]
+    if: needs.git_checks.outputs.num_code_changes > 0
+
+    runs-on: ubuntu-latest
+    container:
+      image: ornladios/adios2:ci-el8-${{ matrix.compiler }}
+      options: --shm-size=1g
+      env:
+        GH_YML_JOBNAME: ${{ matrix.os }}-${{ matrix.gpu_backend }}${{ matrix.compiler }}-${{ matrix.parallel }}
+        GH_YML_BASE_OS: Linux
+        GH_YML_MATRIX_OS: ${{ matrix.os }}
+        GH_YML_MATRIX_COMPILER: ${{ matrix.compiler }}
+        GH_YML_MATRIX_PARALLEL: ${{ matrix.parallel }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [el8]
+        compiler: [icc, oneapi]
+        parallel: [serial, mpi]
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: gha
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          path: source
+      - name: Setup
+        run: gha/scripts/ci/gh-actions/linux-setup.sh
+      - name: Update
+        run: gha/scripts/ci/gh-actions/run.sh update
+      - name: Configure
+        run: gha/scripts/ci/gh-actions/run.sh configure
+      - name: Build
+        run: gha/scripts/ci/gh-actions/run.sh build
+      - name: Test
+        if: ${{ matrix.constrains != 'build_only' }}
```

Comment:
```suggestion
```

We probably do not need this anymore, it was for the GPU builds that could not run tests as we did not have machines with GPUs.

Reply:
Ok, I can remove that.